sdk/core/azure-json/README.md

```diff
@@ -14,7 +14,7 @@ Azure JSON provides shared primitives, abstractions, and helpers for JSON.
 
 #### Include direct dependency
 
-If you want to take dependency on a particular version of the library that is not present in the BOM,
+If you want to take dependency on a particular version of the library that isn't present in the BOM,
```

Comment:
This is needed only when we add `#### Include the BOM file` section above this. Also, if we are not adding the BOM section, the sub-heading `Include direct dependency` is not needed.